src/components/05_pages/NodeForm/index.js

```diff
@@ -121,7 +125,15 @@ class NodeForm extends React.Component {
   };
 
   onSave = () => {
-    this.props.onSave(this.state.entity.data);
+    const missingFields = this.resolveMissingRequiredFields();
+    if (missingFields.length) {
+      this.props.setMessage(
```

Comment:
I think we should have a singular and non singular message here. ```title field_recipe_instruction field_summary are missing.
``` sounds really weird. On top of that, can we extract the title out of the schema?

Reply:
How about using setErrorMessage?